public/index.html

```diff
@@ -4,11 +4,23 @@
   <link rel="stylesheet" type="text/css" href="styles/interface.css">
   <script src="/socket.io/socket.io.js"></script>
   <script src="scripts/main.js"></script>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
```

Comment:
It seems like the right tool for the job. The javascript version of this is a little longer. Whatever, though.